jobs/jobs_utils.py

```diff
@@ -46,3 +52,27 @@ def clone_model(model, **new_values):
     props = {k: v.__get__(model, cls) for k, v in cls._properties.items()} # pylint: disable=protected-access
     props.update(new_values)
     return cls(id=model_id, **props)
+
+
+def get_model_kind(item):
+    """Returns the "kind", a globally unique identifier, of the given item.
+
+    Args:
+        item: base_models.Model|cloud_datastore.Entity|beam_datastore.Entity.
+            The item to inspect.
+
+    Returns:
+        str. The item's kind.
+
+    Raises:
+        TypeError. When the argument is not a model.
+    """
+    if (isinstance(item, base_models.BaseModel) or
+            isinstance(item, type) and issubclass(item, base_models.BaseModel)):
```

Comment:
Suggest adding an inner paren to communicate precedence (whether the "or" or the "and" is done first)